test/unit/materials_managers/test_caching.py

```diff
@@ -88,20 +80,26 @@ def test_custom_partition_name(patch_uuid4):
     assert test.partition_name == custom_partition_name
 
 
-def test_mkp_to_default_cmm(mocker):
-    mocker.patch.object(aws_encryption_sdk.materials_managers.caching, "DefaultCryptoMaterialsManager")
-    mock_mkp = MagicMock(__class__=MasterKeyProvider)
+def test_mkp_to_default_cmm():
+    mkp = ephemeral_raw_aes_master_key()
+
     test = CachingCryptoMaterialsManager(
-        cache=MagicMock(__class__=CryptoMaterialsCache), max_age=10.0, master_key_provider=mock_mkp
+        cache=LocalCryptoMaterialsCache(capacity=10), max_age=10.0, master_key_provider=mkp
```

Comment:
Where does the magic number `10` come from? Should this instead be a declared constant elsewhere?

Reply:
It's not magic, it's just a required parameter. It wasn't there before because I was using mocks before and now I'm actually using the actual class. IMO making it a constant would give it undue gravity; it really doesn't matter what number it is.